test/classwork_test/01_assign_test/types_tests.cpp

```diff
@@ -6,6 +6,15 @@ TEST_CASE("Verify Test Configuration", "verification") {
 	REQUIRE(true == true);
 }
 
+TEST_CASE("test multiply_numbers", "value 50")
+{
+	REQUIRE(multiply_numbers(10) == 50);
+}
+
+TEST_CASE("test multiply_numbers", "value 50")
+{
+	REQUIRE(multiply_numbers(2) == 10);
+}
```

Comment:
Test cases can't have the same name.
"test multiply numbers" for both, your test cases will not run
Change to "test multiply numbers" and "test multiply numbers 1"